### DISCUSSION TITLE
COREHOST_TRACE=1 and COREHOST_TRACEFILE not working as expected

### DISCUSSION THREAD

**Discussion by scottmmendel** [dotnet/runtime#49457] [Q&A]
I am debugging ASP .NetCore 3.1 x64 application that runs from IIS (FDD deployment). An assembly required by the app is generating a System.IO.FileNotFoundException. I was hoping that using COREHOST_TRACE would provide information on the probing used attempting to locate the assembly in question. However, the trace log is missing a lot of basic information that is normally captured when not running under IIS. I have seen some articles on various issues with this utility regarding stderr and stdout. Is there some way to tweak things to capture the probing information I need when IIS launches the application?

**Comment by danmoseley** [comment]
@vitek-karas

**Reply by vitek-karas** [reply]
@jkotalik might know - ASP.NET Core on IIS uses a custom host, so there are differences in behavior.

**Reply by jkotalik** [reply]
Do you see information when you don't have stdout logging enabled? Also, what hosting mode is being used; inprocess or out of process?

**Reply by scottmmendel** [reply]
I was unfamiliar with stdout logging for an IIS application. I enabled "stdoutLogEnabled" and specified a value for "stdoutLogFile" and I was able to successfully capture the full output from the COREHOST tracing utility. Thanks.
